[2023-03-19T13:11:14.258+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transform_bq_data.delete_dataproc_cluster manual__2023-03-19T13:10:51.076599+00:00 [queued]>
[2023-03-19T13:11:14.278+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transform_bq_data.delete_dataproc_cluster manual__2023-03-19T13:10:51.076599+00:00 [queued]>
[2023-03-19T13:11:14.278+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-19T13:11:14.279+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-03-19T13:11:14.280+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-19T13:11:14.308+0000] {taskinstance.py:1303} INFO - Executing <Task(DataprocDeleteClusterOperator): delete_dataproc_cluster> on 2023-03-19 13:10:51.076599+00:00
[2023-03-19T13:11:14.318+0000] {standard_task_runner.py:55} INFO - Started process 2203 to run task
[2023-03-19T13:11:14.332+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'transform_bq_data', 'delete_dataproc_cluster', 'manual__2023-03-19T13:10:51.076599+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/p_spark_processing.py', '--cfg-path', '/tmp/tmp4t6d4ygs']
[2023-03-19T13:11:14.337+0000] {standard_task_runner.py:83} INFO - Job 137: Subtask delete_dataproc_cluster
[2023-03-19T13:11:14.372+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:250: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-03-19T13:11:14.515+0000] {task_command.py:388} INFO - Running <TaskInstance: transform_bq_data.delete_dataproc_cluster manual__2023-03-19T13:10:51.076599+00:00 [running]> on host f1b65b8eb897
[2023-03-19T13:11:14.688+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=transform_bq_data
AIRFLOW_CTX_TASK_ID=delete_dataproc_cluster
AIRFLOW_CTX_EXECUTION_DATE=2023-03-19T13:10:51.076599+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-19T13:10:51.076599+00:00
[2023-03-19T13:11:14.694+0000] {crypto.py:83} WARNING - empty cryptography key - values will not be stored encrypted.
[2023-03-19T13:11:14.695+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-03-19T13:11:14.696+0000] {dataproc.py:901} INFO - Deleting cluster: dataproc-cluster
[2023-03-19T13:11:39.701+0000] {dataproc.py:871} INFO - Cluster deleted.
[2023-03-19T13:11:39.726+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=transform_bq_data, task_id=delete_dataproc_cluster, execution_date=20230319T131051, start_date=20230319T131114, end_date=20230319T131139
[2023-03-19T13:11:39.756+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-19T13:11:39.779+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
